src/components/UI/modal/index.jsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from 'components/UI/modal/modal.module.scss';
-import Backdrop from 'components/UI/backdrop';
+import styles from '@components/UI/modal/modal.module.scss';
+import Backdrop from '@components/UI/backdrop';
 
-const Modal = ({ show, closeModal, children }) => (
+const Modal = ({ show, closeModal, modalChildren, style }) => (
```

Comment:
why `modalChildren`? it's already a Modal component and we do not have to explicitly say the children as `modalChildren`. just `children` should work fine.

Reply:
done changed